lib/matplotlib/table.py

```diff
@@ -482,12 +482,17 @@ def table(ax,
     rows = len(cellText)
     cols = len(cellText[0])
     for row in cellText:
-        assert len(row) == cols
+        if len(row) != cols:
+            msg = "Each row in 'cellText' must have {} columns"
```

Comment:
This formatting does not work with python 2.6, you have to explicitly put in either index or kwargs.